<?xml version="1.0"?>
<launch>
  <node pkg="ig_active_reconstruction_ros" type="simple_viewspace_module" name="baxter_simple_view_module" clear_params="true" output="screen">
    
    <param name="viewspace_file_path" value="$(find baxter_ig_interface)/config/workspace_exploration.txt" />
    
  </node>
</launch>
